packages/doc-gen/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@cadl-lang/doc-gen",
+  "version": "0.1.0",
+  "author": "Microsoft Corporation",
+  "description": "Cadl library for generating cadl docs",
+  "homepage": "https://github.com/Microsoft/cadl",
+  "readme": "https://github.com/Microsoft/cadl/blob/master/README.md",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Microsoft/cadl.git"
+  },
+  "bugs": {
+    "url": "https://github.com/Microsoft/cadl/issues"
+  },
+  "keywords": [
+    "cadl"
+  ],
+  "type": "module",
+  "main": "dist/src/index.js",
+  "exports": {
+    ".": "./dist/src/index.js"
+  },
+  "typesVersions": {
+    "*": {
+      "*": [
+        "./dist/src/index.d.ts"
+      ]
+    }
+  },
+  "cadlMain": "dist/src/index.js",
+  "engines": {
+    "node": ">=16.0.0"
+  },
+  "scripts": {
+    "clean": "rimraf ./dist ./temp",
```

Comment:
@timotheeguerin , do we use a `npm run gendoc` script here to invoke the doc-gen process? What's the invocation step?